packages/main/src/themes/BusyIndicator.css

```diff
@@ -89,6 +89,10 @@
 	flex-direction: column;
 }
 
+:host([_is-over-component]) .ui5-busy-indicator-busy-area {
```

Comment:
use:
:host(:not(:empty)) .ui5-busy-indicator-busy-area {

and remove _isOverComponent property.